recipes/datatable/meta.yaml

```diff
@@ -0,0 +1,65 @@
+{% set name = "datatable" %}
+{% set version = "0.11.0" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/datatable-{{ version }}.tar.gz
+  sha256: 19c602711e00f72e9ae296d8fa742d46da037c2d3a2d254bdf68f817a8da76bb
+
+build:
+  number: 0
+  skip: true  # [py<35 or win]
+  ignore_run_exports:
```

Comment:
This part is not necessary - conda automatically builds one package per python version

Reply:
Does that mean conda will build a package for python 3.4? Given the dependency specified by the upstream project, I had assumed it's better not to publish a package for an unsupported version - is it preferred to err on the side of publishing packages for more Python versions, even if they are not officially supported?

Reply:
I was just focused on the `ignore_run_exports` that part should not be necessary - removing python 3.4 is fine.

Reply:
Oh. I thought the CI check failed without `python` in the `ignore_run_exports`.